### PR TITLE
Adding slogan return

### PR DESCRIPTION
**Pull request by spennyv32**

### CODE REVIEW

**Comment by karlpablo** [comment]
Tomorrow?

File: js_sdk/src/api/config/index.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 const config = require('./config.json');
 
+const temp = process.env.SLOGAN
+
 function getMaintenanceMode() {
   return config.maintenanceMode;
 }
 
+function siteTeaser() {
+  return Base64.decode(temp);
```
Comment (dagumak) [inline js_sdk/src/api/config/index.js:12]:
```suggestion
  return Base64.decode(process.env.SLOGAN);
```

**Comment by windatang** [comment]
we not support for that

**Comment by dong77** [comment]
What is this PR about? I hate to see some keywords in this PR, which only serve to mislead our community.

**Pull request closed without merging**